src/In.ProjectEKA.HipService/Discovery/Matcher/DemographicMatcher.cs

```diff
@@ -0,0 +1,37 @@
+namespace In.ProjectEKA.HipService.Discovery.Matcher
+{
+    using System;
+    using System.Linq.Expressions;
+    using HipLibrary.Matcher;
+    using HipLibrary.Patient.Model;
+
+    public class DemographicMatcher
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a 'protected' constructor or the 'static' keyword to the class declaration.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5359812)